common/authorization/defaultClaimMapper.go

```diff
@@ -0,0 +1,151 @@
+// The MIT License
```

Comment:
this file should be called `common/authorization/jwtClaimMapper.go`?

Reply:
We expect there to be different implementations of jwt claim mappers. This one expects claims to arrive in a token in the specific format: "<namespace>:<read|write|worker|admin>". That's why I called it "default". If it was more generic and supported configurable mappings, then it would qualify to be called jwtClaimMapper IMO. I don't think it does in the current form.

Reply:
Renamed to `defaultJWTClaimMapper`.